### DISCUSSION TITLE
changes made in project specific site.variables and site.overrides not reflecting.

### DISCUSSION THREAD

**Discussion by semanticrelated** [fomantic/Fomantic-UI#1778] [Elements (button, image, label, loader, ...)] [answered]
Hi Experts,

We have "fomantic-ui-less" folder along with its content present under "node_modules" folder on the local system.

In out project front end module, we are having site.variables and site.overrides files where we wanted to change the font for the "default" theme. 

On making changes and after successful build too, the new font isn't reflecting.

Any thoughts on

1] How should we debug this issue.
2] Pointers/suggestions on how to resolve this will be really helpful.

**Comment by lubber-de** [comment]
How did you change the font? i assume you changed `@font` in https://github.com/fomantic/Fomantic-UI-LESS/blob/1d9470b573354b4d16717bc3d8016355de9ecf95/themes/default/globals/site.variables#L11 ?

**Reply by semanticrelated** [reply]
Hi @lubber-de ,

So the file you mentioned, "site.variables" is part of the front end module under the path "ui.frontend\node_modules\fomantic-ui-less\themes\default\globals".

But we have copied over the site.variables and site.overrides to our project front end module and making the changes there, so that we override the default files.. something very similar on the lines of https://github.com/Adobe-Marketing-Cloud/asset-share-commons/tree/develop/ui.frontend.theme.light .

**Comment by lubber-de** [comment, marked as answer]
So you use them in a different theme folder. Then make sure you change the folder name for `site` from `default` to the name of your theme folder in `theme.config` (which you have to create from the example file) https://github.com/fomantic/Fomantic-UI-LESS/blob/1d9470b573354b4d16717bc3d8016355de9ecf95/theme.config.example

**Reply by semanticrelated** [reply]
Hi @lubber-de ,

Based on what you mentioned, followed the exact steps mentioned in https://marekurbanowicz.medium.com/how-to-customize-fomantic-ui-with-less-and-webpack-applicable-to-semantic-ui-too-fbf98a74506c ... but still the same ...

**Reply by semanticrelated** [reply]
My bad... Please give me few minutes.. missed out on adding changes to webpack.config.js as mentioned in the article

**Reply by semanticrelated** [reply]
Hi @lubber-de ,

Phew, finally after several days looks like we got this to work.

Thanks a lot for your inputs.

https://marekurbanowicz.medium.com/how-to-customize-fomantic-ui-with-less-and-webpack-applicable-to-semantic-ui-too-fbf98a74506c article too helped us a lot.

Thanks again for all your help !!!
